tcf_website/templates/reviews/review.html

```diff
@@ -45,7 +45,7 @@ <h6 id="date" class="text-muted">Updated {{ review.created|date:"n/d/y" }}</h6>
                   <div class="review-text-body collapse" id="reviewText{{review.id}}"
                      aria-expanded="false">
                      <!-- Full review text -->
-                    <p class="review-text-full">{{ review.text }}</p>
+                    <div class="review-text-full">{{ review.text|linebreaks }}</div>
```

Comment:
I'm just curious, why div vs. p in this case?

Reply:
@borjainglefernandez 
Sorry for the late reply! I initially kept the `<p>` tag, but margins were added before and after the element, so I replaced it with `<div>`:
- div:
![image](https://user-images.githubusercontent.com/7676354/118323813-29453a00-b53c-11eb-9320-78277aa99925.png)
- p:
![image](https://user-images.githubusercontent.com/7676354/118323898-437f1800-b53c-11eb-9e2f-98ad5e3d6c0a.png)